stm32/support/hex2dfu.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
 #
 # Intel HEX to ST microelectronics DfuSe file converter
 # Copyright (C)2015 Thomas Kindler <mail@t-kindler.de>
```

Comment:
@srsampson I was wondering about the codec2/stm/support Python scripts.  Git log suggested they came from you.  I've never used them and wondered what they are for? Is there any way to test the change to python3 in this PR?

Reply:
Ahh, OK we do use those utilities, and looks like we broke something:
```
cd codec2/stm32/build_stm32/
make clean && make
<snip>
[ 63%] Linking C executable sm1000v5.elf
Creating binary for sm1000v5
Creating dfu file for sm1000v5
  File "/home/david/codec2/stm32/support/hex2dfu.py", line 32
    print "Loading %s..." % args.source
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print("Loading %s..." % args.source)?
```

Reply:
That particular issue should be resolved now.